evals/intent-discovery/harness-capture.eval.ts

```
@@ -0,0 +1,196 @@
+import { existsSync, mkdirSync, readFileSync } from 'node:fs'
```

ESLint

[error] 'node:fs' imported multiple times.

(import/no-duplicates)